sdk/OoyalaSkinSDK/OoyalaSkinSDK/Utils/NSDictionary+Utils.m

```diff
@@ -0,0 +1,63 @@
+//
+//  NSDictionary+Utils.m
+//  OoyalaSkinSDK
+//
+//  Created by Eric Vargas on 11/6/15.
+//  Copyright © 2015 ooyala. All rights reserved.
+//
+
+#import "NSDictionary+Utils.h"
+
+#import <OoyalaSDK/OODebugMode.h>
+#import "OOLocaleHelper.h"
+#import "NSMutableDictionary+Utils.h"
+
+static NSString * const kLocalizableStrings = @"localization";
```

Comment:
can these constants be removed from the OoSkinViewController? Are they there as well?